README.MD

```diff
@@ -19,17 +18,22 @@ A new open source web interface for managing a SaltStack server. Built using van
 ## Quick start using PAM as authentication method
 - Install `salt-api` - this is available in the Salt PPA package which should already been installed if you're using Salt
 - Open the master config /etc/salt/master
-- Find `external_auth` and add a user as follows:
+- Find `external_auth` and configure as following (see the note below!):
 ```
 external_auth:
   pam:
-    desiredSaltGuiUsername:
-      - .*
-      - '@runner'
-      - '@wheel'
+    saltuser:
+      - grains.items
+      - sys.doc
+      - state.apply
+
```

Comment:
this is a straitjacket.
nothing wrong with an explanation (in `PERMISSIONS.md`) about several restricted scenarios.
but the original configuration-set matched the commandline version of `salt`; and that is now gone.
so, whoever really wants to use SaltGUI must do investigation and more configuration to get things done.

Reply:
Yes, maybe you are right: I should leave the default configuration as it was. I will update that.